maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -445,6 +445,22 @@ class Maestro(private val driver: Driver) : AutoCloseable {
         }
     }
 
+    fun findElementFromViewHierarchyWithTimeout(
```

Comment:
Please update `findElementWithTimeout` and make the hierarchy argument optional and when passed use that hierarchy instead of looking it up on the fly